packages/rn-tester/js/components/RNTTitleBar.js

```diff
@@ -43,7 +43,7 @@ const HeaderIOS = ({
             <RNTesterDocumentationURL documentationURL={documentationURL} />
           )}
         </View>
-        {onBack != null && (
+        {!Platform.isTV && onBack != null && (
```

Comment:
This change would also affect Android TV, as `Platform.isTV` is also true there.

Reply:
I think that this shouldn't be the case, as this condition is part of the definition of `HeaderIOS`.

The Android case is handled by a different component, `HeaderAndroid`, which is defined directly below this one."